src/org/robockets/steamworks/Robot.java

```diff
@@ -167,7 +179,7 @@ private void initSmartDashboard() {
 		// CLIMBER //
 		/////////////
 		Robot.climber.initSmartDashboard();
-		SmartDashboard.putData("Climb", new Climb(1));
+		SmartDashboard.putData("Climb", new Climb(-1));
```

Comment:
It shouldn't be possible to change the direction of the climber with a parameter. The climber can only go one direction, and that direction should be a constant that is tucked away where no one else can modify it.

TBH, the `Victor` objects for the climber motors shouldn't even be available in `RobotMap.`